vc_zoom/indico_vc_zoom/client/index.js

```diff
@@ -0,0 +1,29 @@
+// This file is part of the Indico plugins.
+// Copyright (C) 2020 CERN and ENEA
+//
+// The Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License;
+// see the LICENSE file for more details.
+
+import {handleAxiosError, indicoAxios} from 'indico/utils/axios';
+
+document.addEventListener('DOMContentLoaded', async () => {
+  $('.vc-toolbar').dropdown({
+    positioning: {
+      level1: {my: 'right top', at: 'right bottom', offset: '0px 0px'},
+    },
+  });
+
+  document.querySelectorAll('.vc-toolbar .action-make-host').forEach(elem => {
+    elem.addEventListener('click', async () => {
+      let killProgress = IndicoUI.Dialogs.Util.progress();
+      try {
+        await indicoAxios.post(elem.dataset.href);
+        window.location.reload();
+      } catch (error) {
+        handleAxiosError(error);
+      }
+      killProgress();
```

Comment:
I'd only do that inside catch since you reload after a successful POST anyway, so no need to "unblock" the page before that.